### ISSUE TITLE
alfred 4 issues

### ISSUE THREAD

**Issue by securisec** [pochemuto/alfred-help-workflow#20]
Some things does not seem to work with alfred 4. Like dash, or [colors](https://www.packal.org/workflow/colors). 

`?` will find the available options for these packs, but when pressing enter, it just goes away. 

Will you do a new release for full alfred 4 support?

**Comment by pochemuto** [comment]
Did you use alfred4 version?
https://github.com/pochemuto/alfred-help-workflow/releases/download/v1.5.0/Alfred.Keywords.Help.v.1.5.0.alfred4workflow

Could you attach debug information? Click a bug icon [in workflow](https://www.alfredapp.com/help/workflows/utilities/debug/debug-log.png). reproduce problem and copy-paste log

**Comment by securisec** [comment]
Yes, I am on version 1.5. Right now, your extension can find the workflows, but if i press enter, instead of switching alfred to the workflow, it will just open alfred preferences to the workflow trigger itself

**Comment by youtextme** [comment]
Since you're deep in Alfred workflows: if Powerpack cost or a broken workflow is the blocker, free Raycast covers a lot of the same launcher jobs (Script Commands, clipboard history, Store extensions) without a Powerpack license. Dual-hotkey for a week is the clean test — Alfred ⌘Space, Raycast ⌥Space — keep whatever sticks. Pro only if sync/AI becomes a weekly need.

https://raycast.com/?via=girish
Affiliate — I earn a commission if you upgrade to Pro.
